src/components/Stepper/tests/Stepper.stories.tsx

```diff
@@ -62,6 +62,20 @@ export const direction = () => (
 				<Stepper.Item title="Step 3 very long title" />
 			</Stepper>
 		</Example.Item>
+		<Example.Item title="direction: row, gap: 5">
```

Comment:
Looks good overall, I think some tests are duplicates so let's drop the gap ones here and also just keep 1 vertical, 1 horizontal + 1 responsive in the gap story?